src/python/grpcio/tests/unit/framework/interfaces/base/test_cases.py

```diff
@@ -54,13 +54,13 @@ def serialize_request(self, request):
     return request + request
 
   def deserialize_request(self, serialized_request):
-    return serialized_request[:len(serialized_request) / 2]
+    return serialized_request[:int(len(serialized_request) / 2)]
```

Comment:
It looks like ```from __future__ import division``` is preferred and used in most other division instances already, I'll make that change.